src/openMVG/matching_image_collection/Matcher_Regions_AllInMemory.cpp

```diff
@@ -149,7 +151,7 @@ void Template_Matcher(
         ++my_progress_bar;
         if (!vec_FilteredMatches.empty())
         {
-          map_PutativesMatches.insert( make_pair( make_pair(I,J), vec_FilteredMatches ));
+          map_PutativesMatches.insert( make_pair( make_pair(I,J), std::move(vec_FilteredMatches) ));
```

Comment:
I should consider the move instruction in other place:
https://github.com/openMVG/openMVG/blob/develop/src/openMVG/matching_image_collection/Matcher_Regions_AllInMemory.cpp#L152